src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFolder.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Storage;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_Storage
+{
+	[TestClass]
+	public class Given_StorageFolder
+	{
+		[TestMethod]
+		public async Task When_CreateFile_FailIfExists()
+		{
+			var folder = ApplicationData.Current.LocalFolder;
+
+			var filename = $"{Guid.NewGuid()}.txt";
+
+			var file = await folder.CreateFileAsync(filename);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'file'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5685225)